### ISSUE TITLE
Missing support for WARNING and NOTE formatting

### ISSUE THREAD

**Issue by alichtman** [joeyespo/grip#364]
Demo'd here: https://github.com/github-community/community/discussions/16925#discussion-4085374

**Comment by HorridModz** [comment]
Yes, we NEED this! I spent hours integrating this into my workflow with a very robust batch script, only to find that it doesn't support WARNING, NOTE, EXCLAMATION, and emojis (like :facepalm:, for example). Literally the only thing it supports is the color scheme, AFAICT. Supporting themes is nice, but I was under the impression that the whole point of this project was to enable us to view our markown files with Github's markdown flavor and rendering engine.

If the project is just meant to support the color scheme, it would be nice if it could at least *say so* at the top of the README.

**Comment by neilotoole** [comment]
Also, `TIP`!

https://github.com/orgs/community/discussions/16925

**Comment by PathToLife** [comment]
Do we know if this is because of Github's MD rendering API not yet supporting [!NOTE] [!TIP] etc etc?

https://docs.github.com/en/rest/markdown?apiVersion=2022-11-28

For now I've found a workaround which uses the rendered html shown on the Github webpage.
https://gist.github.com/PathToLife/a9c55105484a223d108a4d42e54be4bd

**Comment by alichtman** [comment]
> Do we know if this is because of Github's MD rendering API not yet supporting [!NOTE] [!TIP] etc etc?

Yeah, I think this is the problem.

https://github.com/joeyespo/grip/blob/a3f0ab5c8942ac15cf68f790b80c0a43b3a4c71e/grip/constants.py#L22-L23

https://github.com/joeyespo/grip/blob/a3f0ab5c8942ac15cf68f790b80c0a43b3a4c71e/grip/renderers.py#L41-L84
